lib/server.js

```diff
@@ -3,7 +3,7 @@ import { server as baseServer, routeConfiguringFunction  } from 'appium-base-dri
 import { WindowsDriver } from './driver';
 
 async function startServer(port, host) {
-    let driver = new WindowsDriver(port, host);
+    let driver = new WindowsDriver({port, host});
```

Comment:
hmm I am not sure about this. We only use `{port, host}` here when we are running a standalone windows server. When appium instantiates `WindowsDriver`, it may indeed pass its own set of `args`, but this will contain the `port` and `address` that appium is itself running on. That `port` is something we definitely do _not_ want to start WinAppDriver subprocesses listening on. Unless I'm missing something.